tools/wheels/build-wheels.sh

```diff
@@ -8,7 +8,7 @@ this_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 
 # Skip PyPy, Python2, old Python3 versions, musl, and x86 builds.
-export CIBW_SKIP="pp* cp27-* cp35-* *i686 *musllinux* $CIBW_SKIP"
+export CIBW_SKIP="pp* cp27-* cp35-* cp36-* *i686 *musllinux* $CIBW_SKIP"
```

Comment:
Below, replace 
`so that setup.py finds librdkafka.`
with
`so that cibuildwheel finds librdkafka.`

Reply:
In RELEASE.md there are these lines to change (confluent_kafka.h and pyproject.toml)
```
Commit these changes with a commit-message containing the version:

    $ git commit -m "Version v0.11.4rc1" src/confluent_kafka/src/confluent_kafka.h docs/conf.py pyproject.toml
```

Reply:
Remember to change the `confluent_kafka.c` to `confluent_kafka.h` as it's what was modified just before and needs to be committed.